styles/desktop-styles.css

```diff
@@ -0,0 +1,233 @@
+@media all and (min-width: 768px) {
+  main {
+    background-image: url(../assets/header-llustration-desktop.svg);
```

Comment:
- [ ] I think your app should start the desktop responsiveness from this size fixed here, right? but it is not what is happening, please make sure you fix this
![6](https://github.com/BHObserver/portfolio/assets/77843050/4c7eadb8-35d0-4174-a6cb-e64936b10051)

- [ ] Some of your icon images are not showing too, I think fixing that can be a good idea too
- [ ] In the mobile version, your background image is a bit separate from the right side while it should stick on it as per the design, remember to make all modifications, and in the end, both, mobile and desktop must keep the design template

Reply:
I having some problems with the figma template. There is something I don't understand.